src/utils/mapHelper.js

```diff
@@ -33,12 +33,8 @@ export const mapModel = (model, initialModel, mappingRules = []) => {
 
 		modifiedFields.forEach(modifiedField => {
 			const tempRule = mappingRules.find(rule => rule.modelName === modifiedField);
-			if (tempRule != null) {
-				if (tempRule.transform) {
-					tempRule.transform(model, model[tempRule.modelName].value, result);
-				} else {
-					result[tempRule.domainName] = mapModifiedObj(model[tempRule.modelName]);
-				}
```

Comment:
it is not a breaking change? I do not see when you execute the rule.transform() ? It is already in use in OMS